changelogs/fragments/26-sphinx-antsibull-cfg.yml

```diff
@@ -0,0 +1,8 @@
+minor_changes:
+  - "The ``sphinx-init`` subcommand now also creates an ``antsibull-docs.cfg`` file and moves configuration settings from CLI flags in ``build.sh`` to this configuration file (https://github.com/ansible-community/antsibull-docs/pull/26)."
```

Comment:
excellent, this should make for some interesting possibilities in https://github.com/ansible-community/github-docs-build/ maybe 🤔